azure-pipelines.yml

```diff
@@ -1,126 +1,125 @@
 name: Opdex-API
 trigger:
-- main
+  - main
```

Comment:
Wasn't intended to reformat this file - VSCode did this, although this seems fine